src/realmGeolBasin.ttl

```diff
@@ -39,50 +40,54 @@ sorelph:hasPlanetaryStructure rdf:type owl:ObjectProperty .
 
 ###  http://sweetontology.net/realmGeolBasin/BackArcBasin
 soreagb:BackArcBasin rdf:type owl:Class ;
-                   rdfs:subClassOf soreagb:Basin ;
+                   rdfs:subClassOf soreagb:SedimentaryBasin ;
                    rdfs:label "back arc basin"@en .
 
 
-###  http://sweetontology.net/realmGeolBasin/Basin
-soreagb:Basin rdf:type owl:Class ;
+###  http://sweetontology.net/realmGeolBasin/SedimentaryBasin
+soreagb:SedimentaryBasin rdf:type owl:Class ;
             rdfs:subClassOf soreag:GeologicFeature ,
                             <http://sweetontology.net/reprSpaceGeometry/Depression> ;
-            rdfs:label "basin"@en .
+            dcterms:source <https://doi.org/10.1007/978-3-662-03999-1> ;
+            rdfs:subClassOf soreag:GeologicProvince ;
```

Comment:
SedimentaryBasin sub-class-of Province is not convincing to me. That implies that every individual SedimentaryBasin is also a Province. Is this true @smrgeoinfo ?

Reply:
Yes, every individual sedimentary basin is now regarded by GSQ as a ‘province’. The implication is that sedimentary basins are sedimentary provinces. This was not GSQ’s previous position, where the term ‘province’ was applied only to older sedimentary basins that had been strongly tectonised and/or metamorphosed and whose outlines were indefinite (e.g., see Jell, 2013: https://spaces.hightail.com/space/Cdb1o8ieg1).

Provinces, overall, are now taken to embrace the terms: craton, shield, orogen, orogenic collage, sedimentary basin (sedimentary province, as indicated above), tectonised/metamorphosed province, metallogenic (mineral province), igneous (petrographic) province and large igneous province.

In adopting this approach, GSQ is now in full alignment with Geoscience Australia, in that organisation’s approach to Australian geological provinces: http://www.ga.gov.au/applications/provexplorer/australian-geological-provinces

Reply:
OK - my 1970's geology education may need an update ;-)

Reply:
But I guess that begs a question - while that reflect the consensus in Australia, is it shared throughout the SWEET community?

Reply:
We can look further afield, at the USGS, albeit via Wikipedia:
https://en.wikipedia.org/wiki/Geologic_province#/media/File:World_geologic_provinces.jpg

Reply:
The USGS divide geologic provinces of the World into six main categories (https://earthquake.usgs.gov/data/crust/maps.php): shield (and craton), platform, orogen, basin, large igneous province, and extended crust. I have not referred to the latter and please cf. GA.

Reply:
Is every basin in the Basin and Range of western North America  a province?  That interpretation is not appealing to me.   The B&R province would generally (on this side of the Pacific) be thought of as an extended terrane with many sedimentary basins.

Reply:
I have expressed to John before about the Australian usage of terms and the differences to the way terms are used in the broader community. My view is that Australian geologists seem to like smaller spatial subdivisions when defining orogens or provinces, to an extent that is somewhat decoupled from international usage.
Could there be trouble in paradise?

Reply:
Yes, the USGS regard regions of extended crust as a province (“Extended crust = A province of thinned crust (>50 %) due to extension—e.g. The Basin & Range, North America”): https://earthquake.usgs.gov/data/crust/definitions.html And they give the example of the B & R.
I did not include extended crust in the list of provinces, but, clearly, it must be added, if we decide that this is the way forward, and I believe this should be the case. So, the B&R Province is regarded as an ‘Extended Crust Province’, with its many sedimentary basins. But, every basin in the B&R (Extended Crust) Province need not be considered an individual province. It is all a matter of scale and how the ontology is applied. A common-sense approach must prevail. It seems to me that there can be no ontological Utopia, and that anything that we develop has to cater for, or adapt to, the geological terminology that exists, as varied and confusing as it is. Also, some of the geological terminology may need to change too. All we can do is aim to bring it together with reasonably well defined terms and associated altLabels, as suggested.

Moreover, in taking the B&R a step or two further, the ranges in the province are known as the Great Basin Ranges, and, importantly, the basins are collectively known as the Great Basin, which is largest region of contiguous endorheic drainage in North America. So, herein lies another question: Should the Great Basin itself be identified as a province: a sedimentary basin province, a geographic province, a hydrogeologic (or hydrographic) province? Thus, we would have one province type overlapping another. As for the individual basins or watersheds, the list is long: https://en.wikipedia.org/wiki/List_of_Great_Basin_watersheds

Where do we go from here and how do we/you deal with this (given that the B&R Province, in definition by the USGS, is an existing province)? Are they basins within a basin (The Great Basin)? Or, are they to be regarded as subbasins? That the latter is the case is suggested by the following: https://www.fs.fed.us/rm/boise/research/gis/documents/GBRIHUC4methods3.pdf

But, then you have: 
https://www.nps.gov/grba/planyourvisit/the-great-basin.htm where the following is indicated:
‘The Great Basin includes most of Nevada, half of Utah, and sections of Idaho, Wyoming, Oregon, and California. The term "Great Basin" is slightly misleading; the region is actually made up of many small basins.’

So, terminology and usage thereof are just as confused in the USA as they are in Australia (and undoubtedly elsewhere). (And, certainly, let’s definitely not go down the subprovince route here, which has been the case for metamorphosed/tectonised provinces and subprovinces in Queensland. But, they, nonetheless, exist in the literature.)

The Great Basin, as a present-day, internal-drainage, sedimentary-basin and hydrogeologic system, is comparable in its enormity to the present-day Lake Eyre Basin in central Australia, although the latter has quite different tectonics and lacks basin-and-range structural architecture.

As a side note, my wife and I have explored parts of the B&R some years ago, and, in particular, Death Valley. It was amazing to see geology in action: the springs, dunes, salt deposits, scree deposits and fault scarps. The region has little rainfall, but, when it does rain, it causes a huge amount of erosion and sedimentation. We were totally blown away by then-recent images of car almost completely buried by conglomerate, and a fairly large concrete toilet block (both male and female) completely washed well away from its (concrete) foundations.

To precis: We are stuck with provinces, like it or not. We just need to work out how we deal with them. GA and the USGS consider basins to be provinces. Is the B & R extended crust province sufficient to deal with the basins/subbasins therein? Should the Great Basin also be considered to be a province (sedimentary-basin, hydrogeologic or whatever province), with province types overlapping—is this functional or not? Does the Great Basin comprise basins or subbasins?

Reply:
Perhaps trying to shoehorn the concepts you need into SWEET is the fundamental problem.  Rob Raskin was not a geologist, and that is reflected in the SWEET concept hierarchies.  If I understand the use case, the challenge is to categorize parts of the Earth at a regional scale, and there are different ways of doing this--  deformation process perspective (thrust belt, extended terrane, craton),  sedimentation perspective (foreland basin, forearc basin, intracratonic basin, passive margin embankment), crustal architecture (continent crust, lithosphere, ocean crust), plate tectonics (subduction complex, spreading center, transform margin), magmatic activity (flood basalts, volcanic arc, plutonic arc..).   
 
Seems like the key thing about calling something a 'province' has to do with regional extent, and some unifying tectonic history or processes.

Reply:
Bottom line-- I don't think 'sedimentary basin' should be a kind of province.  Brainstorming here...  Distinction could be: a 'province' is a composite of various features related to a shared geologic/tectonic history in a connected (at the time of the defining history) region, not denoting any particular process.  A sedimentary basin is an individual feature characterized by processes of subsidence and sedimentation.   A sedimentary basin can be partOf a province, but a basin by itself would not be considered a province.

Reply:
Hi Stephen, I thank you and the others for your and their comments—and please except my apologies for my slow reply.

The USGS have used the Glossary definition to define geologic provinces as "any large area or region considered as a whole, all parts of   which are characterized by similar features or by a history differing   significantly from that of adjacent areas", and they divide geologic provinces into six main categories: shield, platform, orogen, basin, large igneous province, and extended crust.

Your comment: "Seems like the key thing about calling something a 'province' has to do with regional extent, and some unifying tectonic history or processes.”

Let me give an example here to demonstrate why sedimentary basins meet your criteria (those cited directly above): I refer to the Sydney Basin - Gunnedah Basin - Bowen Basin system of eastern Australia, with these basins being generally referred to individually, or, if collectively, simply as the   Sydney-Gunnedah-Bowen basin system. Such is the geographic scale of this system (which involved extension/transtension, thermal relaxation, and foreland basin phases), it could be classed as a superbasin. However, it is more massive in geographic coverage than this, as the Bowen Basin was connected to the Galilee Basin to the west (in central Queensland) over the Springsure Shelf, and the Galilee Basin was connected, at times, in its southwest, over the at-times-active Canaway Fault/Ridge, to the Cooper Basin in south-western Queensland and north-eastern South Australia. This   entire system occupied much of eastern, central Australia, and north-eastern South Australia during the Permian – Middle Triassic.

The entire system was not only massive (regionally extensive, a criterion of yours, above), but it had quite-complex, unifying tectonic histories and processes (another criterion of yours for a province). These basins formed largely in the late Pennsylvanian, and were certainly underway by the early Permian. Their initial formation followed the change in rotation of Pangaea, at about 300 Ma, from clockwise to anticlockwise. In eastern Australia, associated dextral/right-lateral transtension (resulting from the anticlockwise rotation) led to the formation of magmatic grabens and half-grabens. This represented the initial birth of the system. Magmatic grabens and half-grabens also formed at the same time in the Northern Hemisphere, such was the global nature of this event, with the loop in the pole path featuring in all apparent polar wander paths from the major continents.

In eastern Australia/eastern Gondwana, following a period of thermal subsidence, continuing anticlockwise rotation gave rise to dextral transpression and commencement of the Hunter-Bowen Orogeny, with its episodic compressional pulses that are marked by unconformity in the sedimentary basins (representing the uplift of half-graben infill) and which have been dated by Ar-Ar step-heating analysis in South Africa, where the orogeny is termed the Cape Orogeny. However, this deformation extended right along the Panthalassan margin of southern Gondwana. Each country has its own name for it, but overall it is embraced by the term, Samfrau Geosyncline (or Orogenic Zone) of DuToit (1937).

These   compressional pulses deformed the eastern margin of Australia, forming a   mountainous thrust belt which gave rise to foreland loading of the   sedimentary basin system. Deformation was more pronounced in the north, where the Bowen Basin was more greatly affected by the deformation, to such an   extent that Jell (2013; Geology of Queensland, GSQ book publication) actually referred the Bowen Basin to the New England Orogen (i.e., he considered the basin to be part of the orogen; not everyone may agree with this). A major compression at the end of the Middle Triassic uplifted the whole system in eastern Australia and terminated sedimentation. This termination extended further afield into the Galilee and Cooper basins.

The tectonic model for the Bowen Basin involves integration of quite-complex tectonic, structural and depositional histories. The more distal Galilee and Cooper basins, as inferred, had comparable histories, although sedimentation in the former began earlier, in its north-east, likely in the latest Mississippian, because of local foreland loading resulting from the formation of a thrust system following mid-Carboniferous Kanimblan orogenesis.

If you cannot advance another criterion or further criteria to exclude sedimentary basins from geologic provinces, then, as the basinal system described above very clearly meets your criteria for a geologic province, then you must accept the fact that sedimentary basins are indeed geologic provinces, a view, indicated above, that is shared by GA and the USGS.

Your comment: "A sedimentary basin is an individual feature characterized by processes of subsidence and sedimentation".

According to Miall (2000), there are three crustal stress environments in which sedimentary basins are created: extensional basins (in which the axis of maximum shear stress is vertical), contractional basins, and shear basins (where, for the latter two, the axis of maximum shear stress is vertical). These different basin types form in various plate-tectonic settings. Tectonism and sedimentation are intimately related (Miall, 2000), even if at least remotely (Dickson, 1975), and, in the latter case, are associated with mantle thermal processes and dynamic topography or epeirogenesis, the influence of which is now known to be of considerable regional significance, involving either thermal expansion or contraction of the crust (Mail, 2000). Downwelling convection (asthenosphere-flow) currents in regions of cold mantle (generally, if not entirely, corresponding with geoid lows) explain the connection between epeirogeny and cratonic-basin development. Ingersol & Busby (1995) listed five subsidence mechanisms that generate accommodation space for sediment, including asthenospheric-flow downwelling, and, although a number of basin classifications exist, all sedimentary basins can be related to their plate-tectonic setting. As such, sedimentary basins are tectonic entities. Even epeirogenesis causes crustal deformation (tectonic deformation), although this is not as readily as apparent as that on a convergent margin, especially in a foreland setting.

Your comment: "I don't see what the distinction is between soreag:TectonicEntity   and soreag:GeologicProvince is anyway".

Yes, Stephen, I agree with you—there is no distinction!! What I did was to exclude from provinces other features that I didn’t classify as provinces.

So, here lies a potential resolution: Even though I have argued above that sedimentary basins are geologic provinces that meet your criteria for being so, let’s not refer to them as provinces, but just as sedimentary basins. This should please both you and Simon.

Sedimentary basins can be classed as tectonic entities, along with other tectonic entitiies that include geologic provinces. I suggest that we eliminate Province as a class and place all under tectonic entities.

Tectonic entities could then embrace:
lithosphere, crust (continental, oceanic) [note that the lithosphere is a tectonic entity as it is subdivided into tectonic plates, which are indisputably tectonic entities; and it will link to other ontologies (layers of Earth), as I think I have seen the inference of this made by Nick?]
tectonic plate
mid-ocean ridge
subduction zone
volcanic arc
present-day continent, ancient continent, supercontinent
craton, shield, platform, supercraton
orogen, orogenic collage
sedimentary basin,subbasin, superbasin
tectonised/metamorphosed province,
igneous (petrographic) province, large igneous province
extended crust province
etc., as other tectonic entities can be further added

Can you accept these proposals? And, can we just define the above provinces under tectonic entities?